tests/acceptance/src/test/java/org/activityinfo/test/steps/common/DatabaseSetupSteps.java

```diff
@@ -209,5 +202,29 @@ public void I_have_created_a_location_in_with_code(String name, String locationT
                 property("code", code));
     }
 
+    @Then("^Location type \"(.*?)\" should appear in tree\\.$")
```

Comment:
Avoid linking the tests to specific UI constructs as they are likely to change over time. Change to: "should be visible"